ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -469,5 +469,5 @@ object KeyValueConsumption {
       .fold(err => throw Err.DecodeError("Party", s"Cannot parse '$s': $err"), identity)
 
   private def correlationId(submitterInfo: DamlSubmitterInfo): String =
-    submitterInfo.getCommandId
+    submitterInfo.getSubmissionId
```

Comment:
@miklos-da We have been back and forth on this point. In the end, we've decided to use the submission id as the error correlation id (to not be confused with the correlationId in the WriteService - which is still the commandId). Are you ok with this change?

Reply:
CC @mziolekda